chainercv/utils/testing/assertions/assert_is_instance_segmentation_dataset.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import six
+
+from chainercv.utils.testing.assertions.assert_is_image import assert_is_image
+
+
+def assert_is_instance_segmentation_dataset(dataset, n_class, n_example=None):
+    """Checks if a dataset satisfies instance segmentation dataset APIs.
+
+    This function checks if a given dataset satisfies instance segmentation
+    dataset APIs or not.
+    If the dataset does not satifiy the APIs, this function raises an
+    :class:`AssertionError`.
+
+    Args:
+        dataset: A dataset to be checked.
+        n_class (int): The number of classes including background.
+        n_example (int): The number of examples to be checked.
+            If this argument is specified, this function picks
+            examples ramdomly and checks them. Otherwise,
+            this function checks all examples.
+
+    """
+
+    assert len(dataset) > 0, 'The length of dataset must be greater than zero.'
+
+    if n_example:
+        for _ in six.moves.range(n_example):
+            i = np.random.randint(0, len(dataset))
+            _check_example(dataset[i], n_class)
+    else:
+        for i in six.moves.range(len(dataset)):
+            _check_example(dataset[i], n_class)
+
+
+def _check_example(example, n_class):
+    assert len(example) >= 4, \
+        'Each example must have at least four elements:' \
+        'img, bbox, mask and label.'
+
+    img, bbox, mask, label = example[:4]
+
+    assert_is_image(img, color=True)
+    _, H, W = img.shape
+    N = bbox.shape[0]
+
+    assert isinstance(label, np.ndarray), \
+        'label must be a numpy.ndarray.'
+    assert bbox.dtype == np.float32, \
+        'The type of bbox must be numpy.float32'
+    assert mask.dtype == bool, \
+        'The type of bbox must be bool'
+    assert label.dtype == np.int32, \
+        'The type of label must be numpy.int32.'
+    assert bbox.shape == (N, 4), \
+        'The shape of bbox must be (N, 4).'
+    assert mask.shape == (N, H, W), \
+        'The shape of mask must be (N, H, W).'
+    assert label.shape == (N,), \
+        'The shape of label must be (N, ).'
+    assert label.min() >= -1 and label.max() < n_class, \
```

Comment:
Will `label.min()` ever be `==-1`?
If not, can you change the minimum allowed value to `0`?